### PR TITLE
Fix the Pipeline runs paging issues caused by BlueOcean plugin

### LINKED ISSUE #3507: When the number of branches is more than 10, no data is returned in the second page.
**Versions Used**
KubeSphere: `dev:latest`

**Preset conditions**
There is a multi-pipeline 'test1' with 12 branchs

**How To Reproduce**
1. Go to pipeline 'test1'
2. Check 'Activity List'
3. Go to page 2

**Expected behavior**
Show two branches

**Actual behavior**
Show no data
image

image

/priority high
/area devops
/cc @kubesphere/sig-devops
/kind bug
/milestone 3.1.0

### PR DESCRIPTION
**Pull request by LinuxSuRen**
**What type of PR is this?**
/kind bug
/area devops

**What this PR does / why we need it**:
The Pipeline runs paging has problems since page 2.

**Which issue(s) this PR fixes**:
Fixes #3507

**Special notes for reviewers**:
The better way might be fixing the Jenkins BlueOcean plugin. But it will take a longer time before we can get a release of it.

**Additional documentation, usage docs, etc.**:

### CODE REVIEW

File: pkg/simple/client/devops/jenkins/pipeline.go [hunk 1]
```diff
@@ -170,34 +171,88 @@ func (p *Pipeline) GetPipelineRun() (*devops.PipelineRun, error) {
 }
 
 func (p *Pipeline) ListPipelineRuns() (*devops.PipelineRunList, error) {
-	res, err := p.Jenkins.SendPureRequest(p.Path, p.HttpParameters)
+	// please don't remove below code lines until blueocen fixed
+	//res, err := p.Jenkins.SendPureRequest(p.Path, p.HttpParameters)
```
Comment (zryfish) [inline pkg/simple/client/devops/jenkins/pipeline.go:175]: Please remove the unused code. If it's commented, it's unnecessary, right?

Comment (LinuxSuRen) [reply pkg/simple/client/devops/jenkins/pipeline.go:175]: As I said in the comment, I prefer to keep it.

Comment (zryfish) [reply pkg/simple/client/devops/jenkins/pipeline.go:175]: It's unprofessional, you can file another PR when blueocean fix it.

Comment (LinuxSuRen) [reply pkg/simple/client/devops/jenkins/pipeline.go:175]: How about moving those code lines to a new function?

**Comment by LinuxSuRen** [comment]
The unit test failure is nothing related with current PR.

File: pkg/simple/client/devops/jenkins/pipeline.go (hunk 1 above)
Comment (zryfish) [reply pkg/simple/client/devops/jenkins/pipeline.go:175]: yeah, that's ok

Comment (LinuxSuRen) [reply pkg/simple/client/devops/jenkins/pipeline.go]: done.

**Pull request merged**